test/Microsoft.AspNetCore.Rewrite.Tests/IISUrlRewrite/InputParserTests.cs

```diff
@@ -45,7 +45,7 @@ public void InputParser_ParseStringWithBackReference(string testString, int expe
         public void EvaluateBackReferenceRule(string testString, string expected)
         {
             var middle = new InputParser().ParseInputString(testString);
-            var result = middle.Evaluate(CreateTestRewriteContext(), CreateTestRuleMatch(), CreateTestCondMatch());
+            var result = middle.Evaluate(CreateTestRewriteContext(), CreateTestRuleMatch().BackReferences, CreateTestCondMatch().BackReferences);
```

Comment:
The `CreateTestRuleMatch` and `CreateTestCondMatch` don't look like they are used for anything other than `BackReferences`, should probably change them